cmd/res/device.virtual.binary.yaml

```diff
@@ -1,3 +1,4 @@
+apiVersion: "v2"
```

Comment:
I don't think this is needed here? We recently removed apiVersion from the sub-DTOs. Same for all device profiles in this PR.